tests/integration/CMakeLists.txt

```diff
@@ -25,21 +25,39 @@
 # THE POSSIBILITY OF SUCH DAMAGE.
 
 
-
+set(COMMON_ARGS "--tstop 100. --celsius 6.3 -mpi")
+set(RING_COMMON_ARGS "--datpath ${CMAKE_CURRENT_SOURCE_DIR}/ring ${COMMON_ARGS}")
+set(RING_GAP_COMMON_ARGS "--datpath ${CMAKE_CURRENT_SOURCE_DIR}/ring_gap ${COMMON_ARGS}")
+set(PERMUTE1_ARGS "--cell-permute 1 --nwarp 8")
+set(PERMUTE2_ARGS "--cell-permute 2 --nwarp 16")
 
 # List of tests with arguments
 set(TEST_CASE_ARGS
-  "ring!--datpath=${CMAKE_CURRENT_SOURCE_DIR}/ring --outpath=${CMAKE_CURRENT_BINARY_DIR}/ring --tstop=100. --celsius=6.3 -mpi"
-  "ring_gap!--datpath=${CMAKE_CURRENT_SOURCE_DIR}/ring_gap --outpath=${CMAKE_CURRENT_BINARY_DIR}/ring_gap --tstop=100. --celsius=6.3 -mpi")
+    "ring!${RING_COMMON_ARGS} --outpath ${CMAKE_CURRENT_BINARY_DIR}/ring"
+    "ring_binqueue!${RING_COMMON_ARGS} --outpath ${CMAKE_CURRENT_BINARY_DIR}/ring_binqueue --binqueue"
+    "ring_multisend!${RING_COMMON_ARGS} --outpath ${CMAKE_CURRENT_BINARY_DIR}/ring_multisend --multisend"
+    "ring_permute1!${RING_COMMON_ARGS} --outpath ${CMAKE_CURRENT_BINARY_DIR}/ring_permute1 ${PERMUTE1_ARGS}"
+    "ring_permute2!${RING_COMMON_ARGS} --outpath ${CMAKE_CURRENT_BINARY_DIR}/ring_permute2 ${PERMUTE2_ARGS}"
 
```

Comment:
This is how I am running/testing ring test with different options (see the arguments).
If any arguments are unnecessary or non-compatible, let me know